completion/git-machete.completion.zsh

```diff
@@ -34,6 +34,12 @@ _git-machete() {
             '(-H --sync-github-prs)'{-H,--sync-github-prs}'[Annotate with GitHub PR numbers and authors where applicable]' \
           && ret=0
           ;;
+        (clean)
+          _arguments \
+            '(--checkout-my-github-prs)'--checkout-my-github-prs'[Checkout your open PRs into local branches]' \
+            '(-y --yes)'{-y,--yes}'[Do not ask for confirmation when deleting unmanaged or untracked branches]' \
```

Comment:
Huh AFAICS now it removes branches without confirmation even when `--yes` is not passed 🤔

Reply:
Hmm, you are right, i have no idea why this happens, will look into it 👀

Reply:
Oh, never mind, it works :P